src/arcaflow_plugin_sdk/schema.py

```diff
@@ -5506,7 +5507,7 @@ def _validate_discriminator(self):
                 raise BadArgumentException(
                     f"the type of object id {set_member.id}'s discriminator"
                     f' field "{self.discriminator_field_name}" does not match'
-                    " OneOfSchema discriminator type; expected"
+                    " discriminated union type; expected"
```

Comment:
How about, "... does not match the discriminator type for the union; expected ..."?

Reply:
did it